Tasks/DotNetCoreCLIV2/README.md

```diff
@@ -35,6 +35,8 @@ Options specific to **dotnet pack** command
 * **Configuration to Package\*:** When using a csproj file this specifies the configuration to package.
 * **Package Folder\*:** Folder where packages will be created. If empty, packages will be created alongside the csproj file.
 * **Do not build\*:** Don't build the project before packing. Corresponds to the --no-build command line parameter.
+* **Include Symbols\*:** Additionally creates symbol nuget packages. Corresponds to the --include-symbols command line parameter.
```

Comment:
after your last updates, it doesn't seem task.loc.json or the resources were updated.  Did you remember to build after your changes to task.json?  also, this readme.md could use a similar 'nuget' => 'NuGet' casing change matching your last commit.

Reply:
Forgot to run build on the last round. Updated now for both items.